db-scheduler-ui/src/main/java/com/github/bekk/dbscheduleruiapi/util/QueryUtils.java

```diff
@@ -53,4 +57,74 @@ public static List<TaskModel> sortTasks(List<TaskModel> tasks, TaskSort sortType
     }
     return tasks;
   }
+
+  public static List<TaskModel> search(List<TaskModel> tasks, String searchTerm) {
+    if (searchTerm == null || searchTerm.trim().isEmpty()) {
+      return tasks;
+    }
+
+    String lowerCaseTerm = searchTerm.toLowerCase();
+
+    return tasks.stream()
+        .filter(
```

Comment:
Jeg synes ofte sånne lange konditions inne i en lambda kan bli litt mer lesbare ved å faktisk skrive de som en "metode". Altså droppe lambdaen, og kjøre old school returns for hver condition